src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -47,6 +49,10 @@ def __init__(self, env_layer, composite_logger, telemetry_writer, status_handler
         # Package manager exit code(s)
         self.apt_exitcode_ok = 0
 
+        # auto OS updates
+        self.find_current_auto_os_updates_settings_cmd = 'sudo cat /etc/apt/apt.conf.d/20auto-upgrades'
```

Comment:
Ensure true sample output (taken from a machine; but scrubbed of machine/user identity) of these commands exist as comments inside the test code.